operator/api/v1alpha1/konflux_types.go

```diff
@@ -57,6 +59,14 @@ type KonfluxUIConfig struct {
 	Spec *KonfluxUISpec `json:"spec,omitempty"`
 }
 
+// NamespaceListerConfig defines the configuration for the namespace-lister component.
+// The Spec field is the runtime configuration passed to the component.
+type NamespaceListerConfig struct {
+	// Spec configures the namespace-lister component.
+	// +optional
+	Spec *KonfluxNamespaceListerSpec `json:"spec,omitempty"`
```

Comment:
I suggest dropping the konflux prefix in this case.
```suggestion
	Spec *NamespaceListerSpec `json:"spec,omitempty"`
```

Reply:
I kept it like this to be consistent with the UI component having both CR name and spec with the Konflux prefix. any reason it should be treated differently here?